python-sgd-regression/sgd_regression.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+from io_helper import io_helper
+from sklearn_to_pfa import sklearn_to_pfa
+
+import logging
+import json
+
+import pandas as pd
+from sklearn.linear_model import SGDRegressor
+import patsy
+import jsonpickle
+
+
+DEFAULT_DOCKER_IMAGE = "python-sgd-regression"
+
+
+def main():
+    # Configure logging
+    logging.basicConfig(level=logging.INFO)
+
+    # Get existing results with partial model if they exist
+    job_result = io_helper.get_results()
+
+    if job_result:
+        # reconstruct estimator from metadata in PFA
+        pfa = json.loads(job_result.data)
+        estimator = deserialize_sklearn_estimator(pfa['metadata'])
+    else:
+        # TODO: SGD-type algorithms require normalized data! how do we do that in incremental learning?
```

Comment:
Normalization in incremental learning might be tricky, I'll take a shot at it tomorrow.